jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/CombinedResource.java

```diff
@@ -343,6 +343,7 @@ public void copyTo(Path destination) throws IOException
                     // use old school stream based copy
                     try (InputStream in = r.newInputStream(); OutputStream out = Files.newOutputStream(pathTo))
                     {
+                        // TODO this is very inefficient as copy() allocates a 64K buffer.
```

Comment:
Not changing the above line to use `IOResources.newInputStream`?

Reply:
This is in `java-util`, so here it's not possible to use `IOResources` which is in `java-io`.